packages/wonder-blocks-link/components/link-core.js

```diff
@@ -9,6 +9,12 @@ import Color, {mix, fade} from "@khanacademy/wonder-blocks-color";
 import type {ClickableHandlers} from "@khanacademy/wonder-blocks-core";
 import type {SharedProps} from "./link.js";
 
+type Target =
+    | "_self"
+    | "_blank"
+    | "_parent"
+    | "_top";
```

Comment:
`LinkCore` isn't exported and nothing is using some of these options.  We should probably just pass `openInNewTab` to `LinkCore` and have it decided what string to use for `target`.

Reply:
I'm fine with that. I was mostly doing this for documentation.